client/src/components/floating-panel-view/floating-panel-view.scss

```diff
@@ -0,0 +1,43 @@
+@import '../../scss/atoms/button';
+@import '../../scss/mixins';
+@import '../../scss/variables';
+@import '../../scss/colors';
+
+.panel-container {
+  @include display-and-animate($default-animation-duration);
+  &.hidden {
+    @include hide-and-animate($default-animation-duration);
+  }
+}
+
+.panel {
+  width: 40vh;
+  height: 90vh;
+  position: fixed;
+  display: flex;
+  background-color: $palette-light;
+  border: solid $palette-medium 1px;
+  box-shadow: 2px 2px 5px $palette-medium;
```

Comment:
design comment, don't make it slide in x axis, so e.g. 0 2px 10px, and try lighter shadow than the border is, otherwise it will just look like a fuzzy border